README.md

```diff
@@ -24,3 +24,7 @@ Please follow [this](https://www.botkube.io/installation/) for complete BotKube
 - **Executor:** Executes BotKube or kubectl command and sends back the result to the Bot interface.
 
 Visit www.botkube.io for Configuration, Usage and Examples.
+
+## Licensing
+BotKube is under [Apache 2.0 license](https://github.com/infracloudio/botkube/blob/develop/LICENSE)
+
```

Comment:
This is already being addressed with https://github.com/infracloudio/botkube/pull/360